<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogM</class>
 <widget class="QDialog" name="DialogM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des matières</string>
  </property>
  <widget class="QPushButton" name="ajouterMatiere">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter une matière</string>
   </property>
  </widget>
  <widget class="QPushButton" name="supprimerMatiere">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer une matière</string>
   </property>
  </widget>
  <widget class="QPushButton" name="afficherMatieres">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Afficher les matières</string>
   </property>
  </widget>
  <widget class="QPushButton" name="retour">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifierMatiere">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier une matière</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
